docs/src/use/configure/migration-guide.md

```diff
@@ -674,6 +674,23 @@ You can see the TypeScript types for the flat config file format in the Definite
 
 You can view the type definitions in the [DefinitelyTyped repository on GitHub](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/eslint/index.d.ts).
 
+## Visual Studio Code Support
+
+By default, the Visual Studio Code ESLint plugin does not look for `eslint.config.js` files. To enable support for the new configuration files, edit your `.vscode/settings.json` file and add the following:
+
+```json
+{
+  /// enable flat config files (eslint.config.*js)
+  "eslint.experimental.useFlatConfig": true
+}
+```
+
+In a future version of the ESLint plugin, you will no longer need to enable this manually.
+
+::: important
+The ESLint team does not maintain the ESLint plugin for Visual Studio Code, and as such, we don't have any control over its release cycle. We are currently waiting on [this issue](https://github.com/microsoft/vscode-eslint/issues/1644) to be resolved.
```

Comment:
[Accessibility] Links that don't provide context in their own text need some kind of additional label or title to explain that context.

```suggestion
The ESLint team does not maintain the ESLint plugin for Visual Studio Code, and as such, we don't have any control over its release cycle. We are currently waiting on [this issue](https://github.com/microsoft/vscode-eslint/issues/1644 "Microsoft/vscode-eslint issue: Enable Flat Config by default") to be resolved.
```

More info: 
* Quick explanation: https://www.webaccessibility.com/resource-library/best-practice/?bestPractice=1301#:~:text=Meaningful%20link%20text%20should%20not,meaning%20in%20the%20surrounding%20content.
* Full WCAG docs: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context